fed/barriers.py

```diff
@@ -201,8 +203,9 @@ def __init__(
         self._party = party
         self._tls_config = tls_config
         self.retry_policy = retry_policy
-        config = fed_config.get_cluster_config()
-        set_max_message_length(config.cross_silo_messages_max_size)
+        self.metadata = fed_config.get_job_config().meta_data
```

Comment:
```suggestion
        self._metadata = fed_config.get_job_config().meta_data
```
Maybe the private member should be used a prefix?